templates/_star_chart_heat_map.html

```diff
@@ -0,0 +1,45 @@
+{% load observations %}
+
+{% for construct, sublevels in dot_matrix.items %}
+
+<div style="overflow-x: scroll;width:100%;">
+    <h4 class="text-align-center" style="margin-bottom: 25px;">{{ construct.abbreviation }}</h4>
+        <table class="table table-bordered">
+            <thead>
+                <tr>
+                    {% for level in construct.sublevels %}
+                    <th scope="col" title="{{ level.description }}" data-toggle="tooltip">
+                        {{ level.short_name }}
+                        {% if level.description|length > 0 %}
+                            <span><i class="fa fa-info-circle"></i></span>
+                        {% endif %}
+                    </th>
+                    {% endfor %}
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    {% for sublevel, observations in sublevels.items %}
+                        {% if observations|length < 3 %}
+                            <td style="height: 60px" class="text-center" bgcolor="#0066FF"></td>
+                        {% elif observations|length < 5 %}
+                            <td style="height: 60px" class="text-center" bgcolor="#99CCFF"></td>
+                        {% elif observations|length == 5 %}
+                            <td style="height: 60px" class="text-center" bgcolor="#FFFFFF"></td>
+                        {% elif observations|length < 8 %}
```

Comment:
I'm not sure it's a good value. Maybe in the future we will choose color dynamically dependently on total number of observation, but now we can left it as it is. Anyway, this is another argument why we shouldn't handle logic related with colors in template.